papyrus-vm/src/papyrus-vm-lib/VirtualMachine.cpp

```diff
@@ -297,7 +298,24 @@ VarValue VirtualMachine::CallMethod(
     if (auto f = nativeFunctions[base][methodName]) {
       auto self = VarValue(selfObj);
       self.SetMetaStackIdHolder(stackData->stackIdHolder);
-      return f(self, arguments);
+      try {
+        return f(self, arguments);
+      } catch (std::exception& e) {
+        std::string methodNameFull;
+        methodNameFull += base;
+        methodNameFull += (base[0] ? "." : "") + std::string(methodName) + "'";
+        spdlog::error("VirtualMachine::CallMethod - Native {} errored with {}",
+                      methodNameFull, e.what());
+        return VarValue::None();
+      } catch (...) {
```

Comment:
Not sure if it's a good idea to do a `...` catch

Reply:
If we get a nonstandard exception, then potential script breakage is our least problem. I think it should lead to a terminate.